__tests__/app.test.js

```diff
@@ -477,6 +477,111 @@ describe('PATCH /api/articles/:article_id', () => {
       });
   });
 });
+describe('PATCH /api/comments/:comment_id', () => {
+  test('PATCH:201 should update votes in DB by comment_id and return correct data type', () => {
+    return request(app)
+      .patch('/api/comments/1').send({ inc_votes: 1 })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toBeInstanceOf(Object);
+        expect(body.comment).toHaveProperty('comment_id');
+        expect(body.comment).toHaveProperty('body');
+        expect(body.comment).toHaveProperty('article_id');
+        expect(body.comment).toHaveProperty('author');
+        expect(body.comment).toHaveProperty('created_at');
+        expect(body.comment).toHaveProperty('votes');
```

Comment:
I think you can check what the number of votes is - primarily because your test is checking that it has been updated :)